.github/actions/ci-container/action.yaml

```diff
@@ -0,0 +1,7 @@
+name: ci-container
```

Comment:
Should we add linux into the name to distinguish the upcoming windows doc?

Reply:
I'll add the copyright, but I will probably just expose an input to the action so that it can have the correct container tag used.  I would like to wait until we know how that will work, I have not done windows builds in a container via github actions, I have only done them on the runner directly.